### DISCUSSION TITLE
I cannot overwrite the getCreateFormAction

### DISCUSSION THREAD

**Discussion by maxacarvalho** [filamentphp/filament#6704] [Help] [answered]
Hi,

I'm trying to overwrite the `getCreateFormAction` method, but it still calls the `create` method from the `CreateRecord` page.

Here's what I'm doing:
```php
protected function getCreateFormAction(): Action
{
    return Action::make('anotherCreate')
        ->label(__('filament::resources/pages/create-record.form.actions.create.label'))
        ->submit('anotherCreate')
        ->keyBindings(['mod+s']);
}
```

I'd expect that the method `anotherCreate` gets called. Instead, the `create` one keeps getting called.

**Comment by pboivin** [comment, marked as answer]
@maxacarvalho Have you tried `->action()` instead of `->submit()`?

**Reply by maxacarvalho** [reply]
Hi @pboivin 
No I didn't. What would be the difference?

**Reply by pboivin** [reply]
It should call your method directly... I'm not 100% sure what `submit()` does in this context.

**Reply by danharrin** [reply]
`submit()` changes the `type="submit"` on the button, which submits the form
